packages/discord.js/src/structures/Message.js

```diff
@@ -355,11 +383,12 @@ class Message extends Base {
     /**
      * Reference data sent in a message that contains ids identifying the referenced message.
      * This can be present in the following types of message:
-     * * Crossposted messages (`MessageFlags.Crossposted`)
-     * * {@link MessageType.ChannelPinnedMessage}
-     * * {@link MessageType.ChannelFollowAdd}
-     * * {@link MessageType.Reply}
-     * * {@link MessageType.ThreadStarterMessage}
+     * - Crossposted messages ({@link MessageFlags.Crossposted})
```

Comment:
This text seems pretty redundant

```suggestion
     * - {@link MessageFlags.Crossposted}
```